stable/coredns/templates/serviceaccount-autoscaler.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.autoscaler.enabled }}
```

Comment:
This probably should check for rbac enabled as well

Reply:
good spot